osu.Game/Screens/OnlinePlay/Multiplayer/Match/MultiplayerMatchSettingsOverlay.cs

```diff
@@ -163,14 +165,6 @@ private void load(OverlayColourProvider colourProvider, OsuColour colours)
                                                                         LengthLimit = 100,
                                                                     },
                                                                 },
-                                                                new Section("Room visibility")
```

Comment:
Was this removal to make room? Might be best to comment out, else I feel this component is going to get lost (also it's still present in playlist but not here - should probably comment out both?)